grammar.js

```diff
@@ -192,6 +194,71 @@ module.exports = grammar({
     normal_statement: ($) => bang_range_command($, 'normal', alias(/ .*/, $.commands)),
     startinsert_statement: ($) => maybe_bang($, tokalias($, 'startinsert')),
     stopinsert_statement: ($) => tokalias($, 'stopinsert'),
+
+    fileencoding: ($) =>
```

Comment:
Maybe name that `encoding` rather?
I am pretty sure we will be able to use that elsewhere...

Reply:
I named it 'fileencoding' just because it is the name of the vim option, but I can change it if you prefer

Reply:
Yeah let's have it as `encoding` rather.

Reply:
There's also `encoding` (as well as  `printencoding`, `makeencoding`, `termencoding`) as well as `scriptencoding` (which is a command though).

Reply:
Yeah but we can't invent a word to describe the encoding, so...